protos/sources/vega/vega.proto

```diff
@@ -7,6 +7,19 @@ import "vega/markets.proto";
 option go_package = "code.vegaprotocol.io/vega/protos/vega";
 
 message StopOrder {
+  message SizeOverrideValue {
+    optional string percentage = 1;
+  }
```

Comment:
Why is that wrapped into another objet? Not a big deal but we end up with an optional message containing an optional value? How about use a oneOff instead?